Sources/Runestone/TextView/TextInput/LayoutManager.swift

```diff
@@ -416,12 +416,20 @@ final class LayoutManager {
 
 // MARK: - UITextInput
 extension LayoutManager {
-    func caretRect(at location: Int) -> CGRect {
+    func caretRect(at location: Int, placeCaretAtNextLineFragmentForLastCharacter: Bool = false) -> CGRect {
```

Comment:
I tested this in combination with `lineSelectionDisplayType = .lineFragment` and the behaviour of the two is not aligned. This method might cause the caret to jump to the next line, but the previous line fragment stays highlighted.

Reply:
Great catch! I had forgotten to test that 🙊 It should be fixed now.

Reply:
Yep, looks good in my testing.